docs/web_ssr/set-up-and-usage/library-context.md

```diff
@@ -39,14 +39,21 @@ To inform the library of the selected store, use:
 window.miam.pos.load(posExternalId: string || number)
 ```
 
+:::warning
+
+If you do not have any store selected, simply call this method with `null` or `undefined` as parameter. **Our library will not work properly if the method is not called.**
```

Comment:
```suggestion
If you do not have any store selected, simply call this method with `null` or `undefined` as parameter. **All features of the library that need a store won't start until miam.pos.load is called.** (Otherwise it can't tell the difference between "the store has not been initialized yet" and "the user has not chosen a store)
```